native/packages/react-native-bpk-component-pagination-dots/src/BpkPaginationDotsIndicator.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import { Animated, Easing, StyleSheet, type AnimatedValue } from 'react-native';
+import PropTypes from 'prop-types';
+import {
+  animationDurationSm,
+  colorGray100,
+  colorWhite,
+  spacingSm,
+  spacingMd,
+  paginationDotSizeSm,
+  paginationDotSizeMd,
+  paginationDotSizeBase,
+} from 'bpk-tokens/tokens/base.react.native';
+
+const styles = StyleSheet.create({
+  indicator: {
+    backgroundColor: colorGray100,
+    borderRadius: spacingMd,
```

Comment:
We have border radius tokens, use `borderRadiusSm` instead?

Reply:
This will be wrong on Android though, one for @willhg

Reply:
I was just using `spacingMd` as a high value to get them to render circular. Have now changed to `borderRadiusPill` which achieves the same thing.